src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/MultiNodeSubsystem/AStarVertex.java

```diff
@@ -0,0 +1,170 @@
+package edu.greenblitz.tobyDetermined.Nodesssss.MultiNodeSubsystem;
+
+import edu.greenblitz.tobyDetermined.Nodesssss.NodeBase;
+import edu.wpi.first.math.Pair;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+
+import static edu.greenblitz.tobyDetermined.RobotMap.NodeSystem.SystemsPos;
+
+public class AStarVertex {
+
+    public static <T> void printPath(LinkedList<T> pathList) {
+        for (int i = 0; i <= pathList.size() - 1; i++) {
```

Comment:
same with the for below

```suggestion
        for (int i = 0; i < pathList.size(); i++) {
```